content/doc/book/system-administration/reverse-proxy-configuration-with-jenkins.adoc

```diff
@@ -0,0 +1,1525 @@
+
+|Server |Host name or address of the proxy server. Example:
+`+proxy.example.com+`
+
+|Port |The port the proxy server listens on. Example: `+8080+`
+
+|User name |If the proxy server requires authentication, enter the name
+of the user here.
+
+|Password |If the proxy server requires authentication, enter the
+password here.
+
+|No Proxy Host |If certain hosts must be contacted directly (without
+going through the proxy), you can specify their names here. You can even
+use `+*+` as a wildcard. Example: `+*jenkins-test*+``+.example.com+`
+
+|Test URL |Enter the *full URL* of a host to use for testing the proxy
+configuration. Example: http://jenkins-ci.org/
+|===
+
+To temporarily disable the proxy configuration, you can add a single
+wildcard character `+*+` on the first line. 
+Erasing the value of *Server* will _discard all values_ and disable 
+the use of a proxy server.
+
+[[JenkinsBehindProxy-Whichpluginssupportaproxyserver]]
+===== Which plugins support a proxy server?
+
+[cols=",,,,",]
+|===
+|*Plugin* |*Since version* |*Supports HTTPS?* |*Supports Exclusions?*
+|*Supports authentication?*
+
+|https://wiki.jenkins.io/display/JENKINS/Fortify+On+Demand+Uploader+Plugin[Fortify
+On Demand Uploader Plugin]
+|https://github.com/jenkinsci/fortify-on-demand-uploader-plugin/releases/tag/fortify-on-demand-uploader-1.08[1.08]
+|image:../../resources/managing/check.svg[(tick)]
+|image:../../resources/managing/check.svg[(tick)]
+|image:../../resources/managing/check.svg[(tick)]
+
+|https://wiki.jenkins.io/display/JENKINS/Git+Plugin[Git Plugin]
+|https://github.com/jenkinsci/git-plugin/releases/tag/git-2.1.0[2.1.0]
+|image:../../resources/managing/check.svg[(tick)]
+|image:../../resources/managing/check.svg[(tick)]
+|image:../../resources/managing/check.svg[(tick)]
+
+|https://wiki.jenkins.io/display/JENKINS/Azure+DevOps+and+Team+Foundation+Server+Plugin[Azure
+DevOps and Team Foundation Server Plugin]
+|https://github.com/jenkinsci/tfs-plugin/releases/tag/tfs-5.1.0[5.1.0]
+|image:../../resources/managing/check.svg[(tick)]
+|image:../../resources/managing/check.svg[(tick)]
+|image:../../resources/managing/check.svg[(tick)]
+
+|https://wiki.jenkins.io/display/JENKINS/Openid+Connect+Authentication+Plugin[Openid
+Connect Authentication Plugin]
+|https://github.com/jenkinsci/oic-auth-plugin/releases/tag/oic-auth-1.4[1.4]
+|image:../../resources/managing/check.svg[(tick)]
+|image:../../resources/managing/check.svg[(tick)]
+|image:../../resources/managing/check.svg[(tick)]
+|===
+
+=== Jenkins says my reverse proxy setup is broken
+
+NOTE: Since Jenkins 1.572 this message can also appear if you don't access
+Jenkins through a reverse proxy: Make sure the Jenkins URL configured in
+the System Configuration matches the URL you're using to access Jenkins.
+
+[[Jenkinssaysmyreverseproxysetupisbroken-Symptoms]]
+==== Symptoms
+
+An error message is displayed in the "Manage Jenkins" page - "It appears
+that your reverse proxy set up is broken"
```

Comment:
Do you mind moving this block under a subsection "Troubleshooting" , as in the screenshot below?

<img width="313" alt="Screenshot 2020-05-29 at 20 24 59" src="https://user-images.githubusercontent.com/1522731/83292519-868c9200-a1ea-11ea-8bf8-221c8bacb255.png">